release/cloud_get_commit.template.json

```diff
@@ -0,0 +1,13 @@
+{
+  "steps": [
+    {
+      "name": "gcr.io/istio-testing/istio-builder:0.5.7",
+      "args": [ "-c", "git clone https://github.com/istio/istio -b \"$_BRANCH\" --depth 1 && istio/release/generate_manifest.sh -a \"$_CHECK_GREEN_SHA_AGE\" -b \"$_BRANCH\" -c \"$_COMMIT\" -r \"$_GCS_RELEASE_TOOLS_PATH\" -v \"$_VERIFY_CONSISTENCY\" " ],
```

Comment:
perhaps rename _GCS_RELEASE_TOOLS_PATH so it is clear that it is build specific.